src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -138,6 +138,26 @@ public void clickShowStats() {
 
     public void clickIndicateMissingResponses() {
         click(indicateMissingResponsesCheckbox);
+        expandPanels();
+    }
+
+    public void expandPanels() {
+        try {
+            // In case the panels are already expanded
```

Comment:
I will have a follow-up PR to change the testing to not expand the panels for most of the testing to save test running time.(Not included in this PR to reduce PR size as not expanding actually changes too many html files)
In that PR I will not include expandPanels into actions that would reload panels.